problem-11.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+// Takes a number as input and reverses its digits
+
+int main()
+{
+    int number, reversedNumber = 0, lastDigit;
+
+    printf("---------Reverse digits of a number---------\n");
+
+    printf("Enter a number: ");
+    scanf("%d", &number);
+
+    int displayNumber = number;

```

Comment:
Generally it is a good practice to not change the input of the program or parameters of the functions because it helps in debugging as we are not changing inputs to the program. For this small program its fine but for large codebases we should avoid this.